client/go/commands/cluster_create.go

```diff
@@ -90,9 +91,14 @@ func (a *CreateNewClusterCommand) Exec(args []string) error {
 		fmt.Println("Error: Bad json response from server")
 		return err
 	}
-
 	//if all is well, print stuff
-	fmt.Fprintf(stdout, "Cluster id: %v", body.Id)
+	if a.options.Json == false {
+		fmt.Fprintf(stdout, "Cluster id: %v", body.Id)
+	}
+	if a.options.Json == true {
+		res, _ := json.Marshal(body)
```

Comment:
Since the body already contains the JSON string, all we need to do is extract it and print it without parsing it.  Here is an example of cluster list:

```
	// Check if we need to print the JSON body
	if a.options.Json {
		// Print JSON body
		s, err := utils.GetStringFromResponse(r)
		if err != nil {
			return err
		}
		fmt.Fprint(stdout, s)
	} else {

		//check json response
		var body glusterfs.ClusterListResponse
		err = utils.GetJsonFromResponse(r, &body)
		if err != nil {
			fmt.Println("Error: Bad json response from server")
			return err
		}

		// Print to user cluster lists
		str := "Clusters: \n"
		for _, cluster := range body.Clusters {
			str += cluster + "\n"
		}
		fmt.Fprintf(stdout, str)
	}
```

This should be done for all commands that get JSON responses